### PR TITLE
feat:added a sorting functin

### PR DESCRIPTION
**Pull request by NIREE23**
Please review my code

### CODE REVIEW

**Comment by pr-th-m** [comment]
- Please add a documentation on explaining the function.

File: array.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+function selectionSort(arr) {
```
Comment (KrithikaSuvarna01) [inline array.js]: Can u Please implement Quick Sort instead of Selection sort

**Review by KrithikaSuvarna01 — COMMENTED** [review]
Can u Please implement Quick Sort instead of Selection sort

File: array.js (hunk 1 above)
Comment (NIREE23) [reply array.js]: done

Comment (hrithik-b01) [reply array.js]: Looks good, please add some documentation at the top(js docs)

**Review by KrithikaSuvarna01 — APPROVED** [review]
Good.

**Review by hrithik-b01 — APPROVED** [review]
**_LGTM_**

**Review by pr-th-m — APPROVED** [review]

**Pull request merged**